src/components/Partners.js

```diff
@@ -1,73 +1,86 @@
-                        <a href="https://www.bongohive.co.zm" target="_blank">
-                            <img width="200" height="200" alt="Bongohive"
-                                src={
-                                    require('../assets/bongohive.png')
-                                }
-                            />
-                        </a>
-                    </div>
-                    <div className = "col-sm"></div>
-
+          <div className="container">
+            <div className="row">
+              <div className="col-sm">
+                <a href="https://www.jetbrains.com" target="_blank">
+                  <img
+                    width="200"
+                    height="200"
+                    alt="JetBrains"
+                    src={require("../assets/jetbrains.png")}
+                  />
+                </a>
+              </div>
+              <div className="col-sm">
+                <a href="https://www.stickermule.com" target="_blank">
+                  <img
+                    width="200"
+                    height="200"
+                    alt="Stickermule"
+                    src={require("../assets/stickermule.svg")}
+                  />
+                </a>
+              </div>
+              <div className="col-sm">
+                <a href="#" target="_blank">
+                  <img
+                    width="200"
+                    height="200"
+                    alt="JetBrains"
+                    src={require("../assets/agoracode.png")}
+                  />
+                </a>
+              </div>
+              <div className="col-sm">
+                <a href="https://www.gitkraken.com" target="_blank">
+                  <img
+                    width="200"
+                    height="200"
+                    alt="GitKraken"
+                    src={require("../assets/gitkraken.svg")}
+                  />
+                </a>
+              </div>
+              <div className="col-sm">
+                <a href="https://www.bongohive.co.zm" target="_blank">
+                  <img
+                    width="200"
+                    height="200"
+                    alt="Bongohive"
+                    src={require("../assets/bongohive.png")}
+                  />
+                </a>
+              </div>
+              {/* Below section is kept for styling purposes */}
+              {/* Section is preset to add new partner */}
+              <div className="col-sm" />
+              <a href="#" target="_blank">
+                <img 
+                    width="200" 
+                    height="200" 
+                    alt="" 
+                    src="" 
+                />
+              </a>
             </div>
           </div>
         </div>
       </div>
+      <div className="center email-header">
+        <h6>
+          If you would like to partner with us, send us an email at{" "}
+          <a href="mailto:info@devcon.co.zm">DevConZM</a>
```

Comment:
Could you change DevConZM to the actual email address so regardless people can copy it

Reply:
No worries this has been changed from the convention name to the email address so that it can be copied. It can also be clicked to open the local mail service from the users computer.